README.md

```diff
@@ -285,6 +285,18 @@ resp, err := rmqc.DeleteShovel("/", "a.shovel")
 
 ```
 
+### Operations on cluster name
+``` go
+// Get cluster name
+cn, err := rmqc.GetClusterName()
+// => ClusterName, err
+
+// Rename cluster
+resp, err := rmqc.SetClusterName(ClusterName{Name: "rabbitmq@rabbit-hole"})
```

Comment:
I initially chose to use a string as parameter, but I though passing a struct is more consistent with the API, since cluster name is set via the request body and not as part of path. I've have no problem to switch back to string.